app/server/lib/scim/index.ts

```diff
@@ -0,0 +1,14 @@
+import * as express from 'express';
+
+import { buildScimRouterv2 } from './v2/ScimV2Api';
+import { HomeDBManager } from 'app/gen-server/lib/homedb/HomeDBManager';
+import { InstallAdmin } from '../InstallAdmin';
```

Comment:
Some new files have relative imports, but we don't like mixing relative and absolute paths in the same file.

Reply:
You're right, I fix that, thanks for notifying!